tests/system/fim/common_tasks/prepare.yml

```diff
@@ -39,6 +39,13 @@
     when:
       - inventory_hostname in groups['linuxagents']
 
+  - name: Create {{ files_fim_testing_linux }} to store outputs in Agents hosts | Linux
+    file:
+      path: "{{ files_fim_testing_linux }}"
+      state: directory
+    when:
+      - inventory_hostname in groups['linuxagents']
```

Comment:
Duplicate this task for Windows hosts please

Reply:
The task has been duplicated for Windows hosts.